hutch_python/plugins/experiment.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from importlib import import_module
+from types import SimpleNamespace
+
+from ..base_plugin import BasePlugin
+from .. import utils
+
+logger = logging.getLogger(__name__)
+
+
+class Plugin(BasePlugin):
```

Comment:
I'm not sure I like having multiple `Plugin` classes. Is there an issue with `ExperimentPlugin` ?

Reply:
It seemed verbose to me to refer to `experiment.ExperimentPlugin` rather than `experiment.Plugin`. It also made it annoying elsewhere to `getattr(module, plugin_name.Title() + 'Plugin')` instead of just `module.Plugin`

Reply:
Seems fair